libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/factories/PendingIntentFactory.kt

```diff
@@ -32,19 +32,19 @@ class PendingIntentFactory(
     private val actionIds: NotificationActionIds,
 ) {
     fun createOpenSessionPendingIntent(sessionId: SessionId): PendingIntent? {
-        return createRoomPendingIntent(sessionId = sessionId, roomId = null, threadId = null)
+        return createRoomPendingIntent(sessionId = sessionId, roomId = null, eventId = null, threadId = null)
     }
 
-    fun createOpenRoomPendingIntent(sessionId: SessionId, roomId: RoomId): PendingIntent? {
-        return createRoomPendingIntent(sessionId = sessionId, roomId = roomId, threadId = null)
+    fun createOpenRoomPendingIntent(sessionId: SessionId, roomId: RoomId, eventId: EventId?): PendingIntent? {
+        return createRoomPendingIntent(sessionId = sessionId, roomId = roomId, eventId = eventId, threadId = null)
     }
 
-    fun createOpenThreadPendingIntent(roomInfo: RoomEventGroupInfo, threadId: ThreadId?): PendingIntent? {
-        return createRoomPendingIntent(sessionId = roomInfo.sessionId, roomId = roomInfo.roomId, threadId = threadId)
+    fun createOpenThreadPendingIntent(roomInfo: RoomEventGroupInfo, eventId: EventId?, threadId: ThreadId): PendingIntent? {
```

Comment:
For parity this method should take `sessionId: SessionId, roomId: RoomId` instead of `roomInfo: RoomEventGroupInfo` (I now that this is not new though)